docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -83,6 +83,31 @@ Modeling Measles in GBD 2017
 
    Make sure to check on measles sequelae as well. Our models so far have not paid much attention to the nonfatal side, but it looks like some of the complications can persist well after someone recovers from measles, so maybe that's important to think about.
 
+GBD Hierarchy
+-------------
+
+**Hierarchy table:**
+
++----------------------------------+------------+-----------+
+| **Cause**                        | **GBD ID** | **Level** |
++----------------------------------+------------+-----------+
+| All causes                       | 294        | 0         |
++----------------------------------+------------+-----------+
+| Communicable, maternal, neonatal | 295        | 1         |
+| and nutritional diseases         |            |           | 
++----------------------------------+------------+-----------+
+| Other infectious diseases        | 961        | 0         |
++----------------------------------+------------+-----------+
+| Measles                          | 341        | 3         |
++----------------------------------+------------+-----------+
+
+- {Measles} = [Moderate Measles, Severe Measles]
+
+**Hierarchy Diagram:**
+
+.. image:: measles_gbd_hierarchy.svg
+   :alt: Measles GBD hierarchy diagram
+	
```

Comment:
I don't think we need a hierarchy diagram for a simple model. For the HIV/TB model it was very helpful, so if others like to see it for simple models, too, please let me know here.

Reply:
I think that it is worth including to avoid any surprises in GBD (just thinking about how anemia due to maternal hemorrhage is under a separate cause from maternal hemorrhage... could be worth including to be sure).

Reply:
I think it's nice to see the hierarchy diagram, particularly for people who aren't yet that familiar with the GBD.  And it could be useful if we want to add complexity to the model later. And for catching surprises like Ali said.

However, for simple models, it might be easier/faster to start with a table, then later put in the effort to replace it with a pretty diagram.

I think the diagram should label the different cause levels (0, 1, 2, 3, 4), and perhaps also indicate which level is the most detailed for the cause, and as James suggested below, use a different box type to visually distinguish the sequelae.

@kiranklc, I like how you show the prevalences of the sequelae in the diagram.